cassandra/cluster.py

```diff
@@ -1124,6 +1131,7 @@ def __init__(self,
                  schema_metadata_page_size=1000,
                  address_translator=None,
                  status_event_refresh_window=2,
+                 experimental_tablet_feature_enabled=False,  #  This field is experimental and may be changed / removed in the future
```

Comment:
Is there any reason for this flag now? I think we can remove it altogether now that we have feature negotation.

Reply:
Feature negotiation is for Scylla to know if it should send the information, but this experimental setting is here, because tablets are still experimental in ScyllaDB and this allows to change the tablets table format, but if drivers start to rely on it, then we cannot. So I am not sure about removing it.

Reply:
We can remove it. Suppose we change the format; then we just bump the feature negotiation string from v1 to v2 and old drivers will no longer negotiate.

Should have thought of it earlier, it makes using tools much easier since we don't have to plumb down the flag everywhere.

Reply:
You are right, I will remove it